APISpecification.yaml

```diff
@@ -573,11 +573,13 @@ components:
           type: string
           description: ISO date string in the YYYY-MM format
           example: "2019-02"
+          nullable: true
         annotation_provider:
           $ref: '#/components/schemas/AnnotationProvider'
```

Comment:
@Mehrnaz-Charkhchi didn't we agree that `assembly_provider` and `annotation_provider` are allowed to be null if there is no `name` information about these providers?